python/tutorials/01-vector-add.py

```diff
@@ -23,6 +23,7 @@
 import triton
 import triton.language as tl
 
+DEVICE = "cuda"
```

Comment:
This will allow the user to change sources at least in one place to run on another device.